### PR TITLE
fix(v2:backend): fix importer

### PR DESCRIPTION
**Pull request by yhwang**
**Description of your changes:**
the input value of `importer_spec` is incorrect. It was component spec. Changed it to importer spec.

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: backend/src/v2/compiler/tektoncompiler/testdata/importer.yaml [hunk 1]
```diff
@@ -83,10 +83,10 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.uid
-          envFrom:
-          - configMapRef:
-              name: metadata-grpc-configmap
```
Comment (Tomcli) [inline backend/src/v2/compiler/tektoncompiler/testdata/importer.yaml:88]: Was this a new standard from kfp v2 or it's just used in this test case? I remember v1 was using env variable for looking up the metadata host.

Comment (yhwang) [reply backend/src/v2/compiler/tektoncompiler/testdata/importer.yaml:88]: this part is not new. that configmap has been there for 3 yrs.

Comment (Tomcli) [reply backend/src/v2/compiler/tektoncompiler/testdata/importer.yaml:88]: I see, this was just under the integration test, but not the official deployment.

**Comment by yhwang** [comment]
thanks, later on, I will build a new image for the api-server and update the manifest to point to that image

**Pull request merged**